project_code/app1/src/app/sample/sample-routing.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { SampleComponent } from './sample.component';
+
+const routes: Routes = [{ path: '', component: SampleComponent }];
```

Comment:
Change the component name with userComponent

Reply:
I've tried to change the component as userComponent which is resulting in error in tsconfig.json

Reply:
UserComponent